### ISSUE TITLE
when pnpm dev , filepath was wrong，there's one more part of the path

### ISSUE THREAD

**Issue by JOYCEQL** [onejs/one#152]
## Description

when `pnpm dev` , filepath was wrong，there's one more part of the path, it seems to have something to do with react-native web

## Steps to Reproduce

1. init a base demo like [初始化教程](https://onestack.dev/docs/installation#5-minute-tutorial)
2. `pnpm dev`

## Expected Behavior

pnpm dev should success

## Screenshots

![image](https://github.com/user-attachments/assets/e452a84d-6760-4cb7-a688-3c7c10d31607)

## Environment Information

- OS: [Windows 11, 22631.4249]
- Version:   one@latest
                  node@22.0.0

**Comment by natew** [comment]
Yes, windows support not ready yet, sorry. We are adding some more clear warnings.
